lib/types.ts

```diff
@@ -2,14 +2,11 @@ interface Token {
   sub?: string
   exp?: number
   iss?: string
+  [claim: string]: unknown
```

Comment:
Why `unknown`? don't we know what could be the types?
isn't it `string` or `number`?